openBCIBoard.js

```diff
@@ -856,180 +1013,174 @@ function OpenBCIFactory() {
      * @author AJ Keller (@pushtheworldllc)
      */
     OpenBCIBoard.prototype._processBytes = function(data) {
+        //console.log(data);
         var sizeOfData = data.byteLength;
         this.bytesIn += sizeOfData; // increment to keep track of how many bytes we are receiving
         if(this.isLookingForKeyInBuffer) { //in a reset state
             var sizeOfSearchBuf = this.searchingBuf.byteLength; // then size in bytes of the buffer we are searching for
             for (var i = 0; i < sizeOfData - (sizeOfSearchBuf - 1); i++) {
                 if (this.searchingBuf.equals(data.slice(i, i + sizeOfSearchBuf))) { // slice a chunk of the buffer to analyze
-                    if (this.searchingBuf.equals(this.moneyBuf)) {
-                        if(this.options.verbose) console.log('Money!');
+                    if (this.searchingBuf.equals(this.searchBuffers.miscStop)) {
+                        if (this.options.verbose) console.log('Money!');
+                        if (this.options.verbose) console.log(data.toString());
                         this.isLookingForKeyInBuffer = false; // critical!!!
                         this.emit('ready'); // tell user they are ready to stream, etc...
+                    } else if (this.searchingBuf.equals(this.searchBuffers.timeSyncStart)) {
+                        this.sync.ntp1 = now();
+                        if(this.options.verbose) console.log('Got time sync request: ' + this.sync.npt1.toFixed(4));
+                        this.sync.ntp2 = now();
+                        this._writeAndDrain('<' + (this.sync.ntp1 * 1000) + (this.sync.ntp2 * 1000));
```

Comment:
This is NOT addition of Numbers, this is concatenation of strings because the first character is `<` using the `+` casts the other argument as a string.